packages/middleware-apply-body-checksum/package.json

```diff
@@ -47,6 +48,7 @@
     "directory": "packages/middleware-apply-body-checksum"
   },
   "devDependencies": {
+    "@smithy/util-test": "workspace:^",
```

Comment:
The private, `@smithy/util-test` package is added here so it will be built before tests are run.

Will this be confusing when these packages are published?